src/javascript/app/hooks/growthbook.js

```diff
@@ -0,0 +1,57 @@
+// growthbook.js
+import { useState, useEffect } from 'react';
+
+import { Analytics } from '@deriv-com/analytics';
+
+export const useIsGrowthbookIsLoaded = () => {
+    const [isGBLoaded, setIsGBLoaded] = useState(false);
+
+    useEffect(() => {
+        let checksCounter = 0;
+        const analyticsInterval = setInterval(() => {
+            // Check if the analytics instance is available for 10 seconds before setting the feature flag value
+            if (checksCounter > 20) {
```

Comment:
not sure why is this required, but i see that this is carried over implementation.

Reply:
No this hook file is no longer needed, will remove it for now 😅